lib/Intersection.js

```diff
@@ -176,8 +176,8 @@ Intersection.intersectBezier2Bezier2 = function(a1, a2, a3, b1, b2, b3) {
     c20 = new Point2D(b1.x, b1.y);
 
     // bezout
-    var a = c12.x*c11.y - c11.x*c12.y;
-    var b = c22.x*c11.y - c11.x*c22.y;
+    a = c12.x*c11.y - c11.x*c12.y;
```

Comment:
As an aside to this, the vast majority of this code was written 15 years ago. I'd actually love to drop `var` and switch to `const` and `let` in general

Reply:
Sure; I thought for this PR I'd just get the official ESLint recommended rules going. `prefer-const` and `no-var` are two rules which would enforce this and FWIW, the config of mine I mentioned already enforces these.